agent/job_runner_test.go

```diff
@@ -37,3 +37,35 @@ func TestValidateRepository(t *testing.T) {
 	err = validateRepository(allowedRepos, pipelineRepo)
 	require.NoError(t, err)
 }
+
+func TestAllowedEnvironmentVariables(t *testing.T) {
+	// Allow list without regex
+	var envVarsWithoutRegex = map[string]string{
```

Comment:
```suggestion
	envVarsWithoutRegex := map[string]string{
```

This is more idomatic go.